charts/ai-software-templates/chatbot/values.yaml

```
@@ -12,7 +12,7 @@ model:
 
 gitops:
   gitSecretName: "github-secrets"
-  gitSecretKeyToken: "GITHUB_TOKEN"
+  gitSecretToken: "GITHUB_TOKEN"
```

I think here we should only update the `GITHUB_TOKEN` to `password` right? Because we have created the secret in the pipelines setup:
 ```suggestion
  gitSecretKeyToken: "password"
```

---

This might need to be updated on the `charts/ai-software-templates/chatbot/README.md` and `charts/ai-software-templates/chatbot/values.schema.json` too.

---

yep I had it that way in my `private-values.yaml` I used for testing but didn't update the `values.yaml`.

the schema json is correct:  https://github.com/redhat-ai-dev/ai-lab-helm-charts/blob/57174568b87260207079a2a2f801e89c59d4a05d/charts/ai-software-templates/chatbot/values.schema.json#L58-L62

the readme had the correct `gitSecretKeyToken` but the wrong default

will address in next push